python/sglang/srt/model_executor/model_runner.py

```diff
@@ -536,6 +542,7 @@ def init_memory_pool(
             max_context_len=self.model_config.context_len + 4,
             device=self.device,
             use_records=False,
+            memory_saver_adapter=self.memory_saver_adapter,
```

Comment:
Can we do something similar to how we handle `get_model`, so we do not need to pass `memory_saver_adapter` as an argument to all kinds of memory pools.
```
        with self.memory_saver_adapter.region():
            self.req_to_token_pool = ReqToTokenPool(
```